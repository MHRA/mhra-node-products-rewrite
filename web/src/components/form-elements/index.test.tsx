import renderer from 'react-test-renderer';
import React from 'react';
import { Button, Checkbox } from './index';

describe('Button', () => {
  it('should render', () => {
    const component = renderer
      .create(<Button type="submit" value="val" />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});

describe('Checkbox', () => {
  it('should render', () => {
    const changeHandler = () => {
      return null;
    };
    const component = renderer
      .create(
        <Checkbox
          value="val"
          name="agree"
          id="agree-checkbox"
          onChange={changeHandler}
        />,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
