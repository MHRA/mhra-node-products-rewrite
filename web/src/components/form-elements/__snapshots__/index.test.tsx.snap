// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render 1`] = `
<input
  className="sc-gsnTZi jmRmnt"
  type="submit"
  value="val"
/>
`;

exports[`Checkbox should render 1`] = `
<input
  className="sc-bczRLJ ldzjBD"
  id="agree-checkbox"
  name="agree"
  onChange={[Function]}
  type="checkbox"
  value="val"
/>
`;
